### DISCUSSION TITLE
Position is missing in the dataframe

### DISCUSSION THREAD

**Discussion by w-i-z-a-r-d** [theOehrly/Fast-F1#375] [Q&A]
hi, when listing the content of the laps dataframe, the position is missing and can't use this feature. can you please check?

**Comment by Casper-Guo** [comment]
The `position` column is only available starting at version 3.0.0 so it might be because your local installation of fastf1 is not up to date.

Otherwise, it would be helpful if you can provide the code segment that demonstrates this issue.

**Reply by Suvashan1** [reply]
Hi. I am also unable to access the "position" column and I have v3.1.2 installed.

**Reply by Casper-Guo** [reply]
Can you please send a minimal code sample that demonstrates this issue?

**Reply by Suvashan1** [reply]
I tried running the position vs. lap (https://docs.fastf1.dev/examples_gallery/plot_position_changes.html#sphx-glr-examples-gallery-plot-position-changes-py). This is where I picked up the issue.

**Reply by theOehrly** [reply]
@Suvashan1 This code works perfectly fine for me on the latest version of FastF1. (I downloaded the python file that is linked at the bottom of the example for verification.)

Can you please post the full traceback of the error that you are getting? Are you running exactly that code?

Please ensure that you actually have the latest version of FastF1 installed. For this, run the following code
```
import fastf1
print(fastf1.__version__)
```
It should print "3.1.2".

**Reply by Suvashan1** [reply]
Thank you for this, and apologies. Seems like I was using an older version of python, which in turn uses an older version of fastf1. However, when importing fastf1, I get the following error:

_pandas_extensions    WARNING    Import of pandas internals failed
Traceback (most recent call last):
  File "C:\Users\Suvashan\AppData\Local\Programs\Python\Python39\lib\site-packages\fastf1\internals\pandas_extensions.py", line 9, in <module>
    from pandas.core.internals.construction import \
ImportError: cannot import name 'BlockPlacement' from 'pandas.core.internals.construction' (C:\Users\Suvashan\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\core\internals\construction.py)_

I can run the code for the positions vs lap, but I also get a warning of sorts, saying _pandas_extensions    WARNING    Falling back to slow data frame creation!_. It seems obvious both warnings/errors are related. Any way to resolve this?

**Reply by theOehrly** [reply]
> Thank you for this, and apologies. Seems like I was using an older version of python, which in turn uses an older version of fastf1. However, when importing fastf1, I get the following error:
> 
> _pandas_extensions    WARNING    Import of pandas internals failed
> Traceback (most recent call last):
>   File "C:\Users\Suvashan\AppData\Local\Programs\Python\Python39\lib\site-packages\fastf1\internals\pandas_extensions.py", line 9, in <module>
>     from pandas.core.internals.construction import \
> ImportError: cannot import name 'BlockPlacement' from 'pandas.core.internals.construction' (C:\Users\Suvashan\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\core\internals\construction.py)_
> 
> I can run the code for the positions vs lap, but I also get a warning of sorts, saying _pandas_extensions    WARNING    Falling back to slow data frame creation!_. It seems obvious both warnings/errors are related. Any way to resolve this?

The warning about pandas extension is really just a warning. It will not have any consequences except for noticeably slower data loading.

I suppose that you are using an old version of pandas as well. Try updating pandas and the warning should go away.
